### PR TITLE
test/include test app signal app

### PR DESCRIPTION
**Pull request by soofstad**
Two bugfixes:
- GenericList plugin now uses correct "[]" syntax to request items from a list
- "useRecipe" now includes "initialUiRecipe" in the search for a recipe specified by a name

Include the test application "signalApp".

## Issues related to this change
none

### CODE REVIEW

**Comment by KristianKjerstad** [comment]
Could not test the new app

image


seems to be issue with paths. 

Tried latest dm-cli and the latest dmss version as defined in the docker-compose file in this PR.

File: example/app/data/DemoDataSource/apps/MySignalApp/models/CaseProxy.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "CaseProxy",
+  "type": "CORE:Blueprint",
+  "extends": ["CORE:NamedEntity"],
+  "description": "",
+  "attributes": [
+    {
+      "name": "type",
+      "attributeType": "string",
+      "type": "CORE:BlueprintAttribute"
```
Comment (KristianKjerstad) [inline example/app/data/DemoDataSource/apps/MySignalApp/models/CaseProxy.json:10]: is type needed in attributes list if we extend from NamedEntity?

File: example/app/data/DemoDataSource/apps/MySignalApp/models/plugins/tabs/AttributeSelectorPluginConfig.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
[the remark this answers is not in the document]
Comment (soofstad) [reply example/app/data/DemoDataSource/apps/MySignalApp/models/plugins/tabs/AttributeSelectorPluginConfig.json]: removed plugins blueprints folder :)

File: example/app/data/DemoDataSource/apps/MySignalApp/models/CaseProxy.json (hunk 1 above)
Comment (soofstad) [reply example/app/data/DemoDataSource/apps/MySignalApp/models/CaseProxy.json:10]: No. Removed

**Pull request merged**